$OpenBSD: patch-cmake_Unix_cmake,v 1.5 2018/07/03 05:06:54 rsadowski Exp $

- Avoid forced linkage with libstdc++.
- drop CMake calls to QT5_WRAP_CPP()
-- https://github.com/nomacs/nomacs/pull/187

Index: cmake/Unix.cmake
--- cmake/Unix.cmake.orig
+++ cmake/Unix.cmake
@@ -66,6 +66,7 @@ if(ENABLE_RAW)
 	if(NOT LIBRAW_FOUND)
 		message(FATAL_ERROR "libraw not found. It's mandatory when used with ENABLE_RAW enabled")
 	else()
+		list(REMOVE_ITEM LIBRAW_LIBRARIES stdc++)
 		add_definitions(-DWITH_LIBRAW)
 	endif()
 endif(ENABLE_RAW)
